pkgs/development/libraries/discord-gamesdk/default.nix

```
@@ -14,6 +15,10 @@ stdenv.mkDerivation rec {
     stripRoot = false;
   };
 
+  outputs = [ "out" "dev" ];
+
+  buildInputs = [ gcc.cc.libgcc ];
```

```suggestion
  buildInputs = [ (stdenv.cc.cc.libgcc or null) ];
```
`buildInputs` should be below `nativeBuildInputs`

---

Done, except wdym "remember to remove the input"? Which input, where?

---

Done.